File: 241022/출석 부르기/calling-attendance.js
const fs = require("fs");
let number = Number(fs.readFileSync(0).toString().trim());

if (number === 1) {
    console.log("John");
} else if (number === 2) {
    console.log("Tom");
} else if (number === 3) {
    console.log("Paul");
} else {
    console.log("Vacancy");
}

File: 241022/출석 부르기/calling-attendance.cpp
#include <iostream>
using namespace std;

int main() {
    int number;
    cin >> number;

    if (number == 1) {
        cout << "John";
    } else if (number == 2) {
        cout << "Tom";
    } else if (number == 3) {
        cout << "Paul";
    } else {
        cout << "Vacancy";
    }
    return 0;
}